Framework/NexusCpp/src/NeXusFile.cpp

```diff
@@ -45,68 +45,38 @@ static vector<int64_t> toInt64(const vector<int> &small_v) {
 
 namespace NeXus {
 
+std::ostream &operator<<(std::ostream &stm, NXstatus status) { return stm << static_cast<int>(status); }
```

Comment:
I think `status` and `type` could be marked `const` so this could be used in more contexts.